digit-health-services/libraries/services-common/libraries.iml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove this file. It should not be there in your commit. Any commit.

Reply:
removed.